azure/template.json

```diff
@@ -201,6 +361,18 @@
                                 {
                                     "name": "Cdn:Url",
                                     "value": "[parameters('cdnUrl')]"
+                                },
+                                {
+                                    "name": "LoggingRedisConnectionString",
```

Comment:
This already exists in the app serviceConnectionStrings section. I think keep this one but remove the other reference. Do this as well for the employer app service